main.go

```diff
@@ -109,11 +110,15 @@ func main() {
 	signal.Notify(closedown, os.Interrupt)
 	signal.Notify(closedown, syscall.SIGTERM)
 
-	// Open a web socket to the server TODO!! This won't always be necessary if we're not sending metrics & calculating demand locally
-	ws, err := utils.InitWebSocket(st.microscalingAPI)
-	if err != nil {
-		log.Errorf("Failed to open web socket: %v", err)
-		return
+	var ws *websocket.Conn
+
+	// Open a web socket to the server if needed.
+	if st.demandEngine != "LOCAL" && st.monitorTypes != "none" {
+		ws, err = utils.InitWebSocket(st.microscalingAPI)
+		if err != nil {
+			log.Errorf("Failed to open web socket: %v", err)
+			return
+		}
```

Comment:
This isn't very pretty but without it a websocket connection is opened to `app.microscaling.com` which doesn't exist anymore.

Reply:
I wonder if the new import is also going to help with #44 ?